idea_town/frontend/static-src/app/views/details-view.js

```diff
@@ -0,0 +1,83 @@
+import Model from 'ampersand-model';
+import Collection from 'ampersand-collection';
+import CollectionRenderer from 'ampersand-collection-view';
+
+import BaseView from './base-view';
+import template from '../templates/details-view';
+import ContributorView from './contributor-view';
+
+const Contributors = Collection.extend({
+  model: Model.extend({
+    extraProperties: 'allow'
+  })
+});
+
+export default BaseView.extend({
+  _template: template,
+
+  bindings: {
+    'model.modified_date': {
+      type: function(el, value) {
+        return new Date(value);
+      },
+      hook: 'modified-date'
+    },
+
+    'model.description': {
+      hook: 'modified-date'
+    },
+
+    'model.contribute_url': [{
+      hook: 'contribute-url'
+    },
+    {
+      type: 'attribute',
+      name: 'href',
+      hook: 'contribute-url'
+    }],
+
+    'model.measurements': [{
+      type: 'innerHTML',
+      hook: 'measurements-html'
+    },
+    {
+      type: 'toggle',
+      hook: 'measurements-container'
+    }],
+
+    'model.changelog_url': {
+      type: 'attribute',
+      name: 'href',
+      hook: 'changelog-url'
+    },
+
+    'model.thumbnail': {
+      type: 'attribute',
+      name: 'src',
+      hook: 'thumbnail'
+    },
+
+    'model.details.image': {
+      type: 'attribute',
+      name: 'src',
+      hook: 'detail-image'
+    },
+
+    'model.details.copy': {
+      type: 'text',
+      hook: 'detail-copy'
+    },
+
+    'model.details.headline': {
+      hook: 'detail-headline'
+    }
+  },
+
+  afterRender() {
```

Comment:
[`renderCollection()` in ampersand-view](https://github.com/AmpersandJS/ampersand-view/blob/master/ampersand-view.js#L363) would be handy here. It takes care of building a `CollectionView` and rendering it as a subview. [It works something like this](https://github.com/lmorchard/c25k-web/blob/master/src/js/views/WorkoutList.js#L12).